get.yml

```diff
@@ -0,0 +1,29 @@
+config:
+  target: 'http://localhost:8081'
+  processor: "./artilleryprocessor.js"
+  phases:
+    - duration: 60
+      arrivalRate: 1000
+  defaults:
+    headers:
+scenarios:
+- name: "Look at houses 1"
+  flow:
```

Comment:
don't you need to add a weight for this one?

Reply:
The default weight is 1

Reply:
Default weight is one